test/foo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "foo",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
This was to make sure `metadata` is reading the version correctly. It somehow gave me more confidence than `t.equal(meta.version, '0.0.0');`, but maybe `1.2.3` would be even better.